cmd/openshift-install/create.go

```diff
@@ -80,7 +80,7 @@ var (
 			// FIXME: add longer descriptions for our commands with examples for better UX.
 			// Long:  "",
 		},
-		assets: []asset.WritableAsset{&bootstrap.Bootstrap{}, &machine.Master{}, &machine.Worker{}},
+		assets: []asset.WritableAsset{&bootstrap.Bootstrap{}, &machine.Master{}, &machine.Worker{}, &kubeconfig.Admin{}},
```

Comment:
This seems fine to me.  We'll probably stuff `manifest.json` in here too once we have it pulled out into its own asset structure.  But I'm fine handling *that* in follow-up work.